core/src/main/java/org/apache/carbondata/core/indexstore/DataMapType.java

```diff
@@ -20,7 +20,7 @@
  * Datamap type
  */
 public enum DataMapType {
-  BLOCKLET("org.apache.carbondata.datamap.BlockletDataMap");
+  BLOCKLET("org.apache.carbondata.core.indexstore.blockletindex.BlockletTableMap");
```

Comment:
please keep a Class Object in the `DataMapStoreManager` instead of using this enum